nntrainer/dataset/func_data_producer.cpp

```diff
@@ -80,4 +80,8 @@ FuncDataProducer::finalize(const std::vector<TensorDim> &input_dims,
     return last;
   };
 }
+
+void FuncDataProducer::exportTo(Exporter &exporter,
+                                const ExportMethods &method) const {}
```

Comment:
how about make it throw - as this is not serializable, this call shouldn't be made.

Reply:
Good idea :)

Reply:
That's good idea, but in this case we have to have try catch inside normal control flow, so I'll keep query by isSerializable.